modules/update-common-terms.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding-openshift-updates.adoc
+
+:_content-type: REFERENCE
+[id="update-common-terms_{context}"]
+
+= Common Terms
+
+Control Plane:: The _control plane_, which is composed of control plane machines, manages the {product-title} cluster. The control plane machines manage workloads on the compute machines, which are also known as worker machines. The cluster itself manages all updates to the machines by the actions of the Cluster Version Operator (CVO), the Machine Config Operator (MCO), and a set of individual Operators.
+
+Cluster Version Operator:: The _Cluster Version Operator_ (CVO) checks with the OpenShift Update Service to view valid updates and update paths based on your current component versions and information in the graph.
+
+Machine Config Operator:: The _Machine Config Operator_ (MCO) is a cluster-level operator that manages the operating system and keeps the cluster up to date and configured. Through the MCO, platform administrators can configure and update systemd, CRI-O and Kubelet, the kernel, NetworkManager, and other system features on the nodes.
++
+For more information, see link:https://docs.openshift.com/container-platform/latest/post_installation_configuration/machine-configuration-tasks.html#machine-config-overview-post-install-machine-configuration-tasks[Machine config overview].
```

Comment:
I am not familiar with doc tooling, but it feels a bit weird to have this link that includes a hosting location, instead of something more [relative][1] like `../post_installation_configuration/machine-configuration-tasks.html#machine-config-overview-post-install-machine-configuration-tasks` or some such.

[1]: https://datatracker.ietf.org/doc/html/rfc3986/#section-4.2

Reply:
@wking this was to get around the restriction of not being able to link within a module. I don't even know (right now) if this will be allowed in terms of peer review.